BoardingPassExample/Onboarding Demo/DemoCompatibility.swift

```diff
@@ -0,0 +1,101 @@
+//
```

Comment:
#? do we need Swift 2.3 cross compatibility in the demo, I'd much rather see a Swift 2.3 demo and a Swift 3 demo